<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PC322939" name="com.pulselms.webportal.test.LoginTest" tests="88" failures="6" timestamp="9 Feb 2016 08:05:40 GMT" time="2691.817" errors="0">
  <testcase name="pearsonPulseLoginTest" time="26.841" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="19.935" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="62.394" classname="com.pulselms.webportal.test.LoginTest">
    <failure type="com.pulselms.app.framework.BrowserBotException" message="Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id=&#039;main&#039;]/div/div[2]/div[1]/h3
Build info: version: &#039;2.47.1&#039;, revision: &#039;unknown&#039;, time: &#039;2015-07-30 11:02:44&#039;
System info: host: &#039;PC322939&#039;, ip: &#039;10.25.154.48&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_71&#039;
Driver info: driver.version: unknown">
      <![CDATA[com.pulselms.app.framework.BrowserBotException: Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id='main']/div/div[2]/div[1]/h3
Build info: version: '2.47.1', revision: 'unknown', time: '2015-07-30 11:02:44'
System info: host: 'PC322939', ip: '10.25.154.48', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_71'
Driver info: driver.version: unknown
	at com.pulselms.app.framework.WebDriverBase.wait(WebDriverBase.java:89)
	at com.pulselms.webportal.page.DashboardPage.waitForDashboardPageToLoad(DashboardPage.java:29)
	at com.pulselms.webportal.test.LoginTest.validateLogin(LoginTest.java:68)
	at com.pulselms.webportal.test.LoginTest.pearsonPulseLoginTest(LoginTest.java:46)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- pearsonPulseLoginTest -->
  <testcase name="pearsonPulseLoginTest" time="59.173" classname="com.pulselms.webportal.test.LoginTest">
    <failure type="com.pulselms.app.framework.BrowserBotException" message="Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id=&#039;main&#039;]/div/div[2]/div[1]/h3
Build info: version: &#039;2.47.1&#039;, revision: &#039;unknown&#039;, time: &#039;2015-07-30 11:02:44&#039;
System info: host: &#039;PC322939&#039;, ip: &#039;10.25.154.48&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_71&#039;
Driver info: driver.version: unknown">
      <![CDATA[com.pulselms.app.framework.BrowserBotException: Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id='main']/div/div[2]/div[1]/h3
Build info: version: '2.47.1', revision: 'unknown', time: '2015-07-30 11:02:44'
System info: host: 'PC322939', ip: '10.25.154.48', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_71'
Driver info: driver.version: unknown
	at com.pulselms.app.framework.WebDriverBase.wait(WebDriverBase.java:89)
	at com.pulselms.webportal.page.DashboardPage.waitForDashboardPageToLoad(DashboardPage.java:29)
	at com.pulselms.webportal.test.LoginTest.validateLogin(LoginTest.java:68)
	at com.pulselms.webportal.test.LoginTest.pearsonPulseLoginTest(LoginTest.java:46)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- pearsonPulseLoginTest -->
  <testcase name="pearsonPulseLoginTest" time="28.244" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.007" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.642" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="25.951" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.244" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="20.436" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="23.545" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.154" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.789" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.276" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.419" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.691" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="21.795" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.068" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="26.508" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="21.155" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.425" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="34.310" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.239" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.668" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.281" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.120" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.400" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.450" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.897" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.255" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="23.489" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="23.246" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.005" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="61.518" classname="com.pulselms.webportal.test.LoginTest">
    <failure type="com.pulselms.app.framework.BrowserBotException" message="Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id=&#039;main&#039;]/div/div[2]/div[1]/h3
Build info: version: &#039;2.47.1&#039;, revision: &#039;unknown&#039;, time: &#039;2015-07-30 11:02:44&#039;
System info: host: &#039;PC322939&#039;, ip: &#039;10.25.154.48&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_71&#039;
Driver info: driver.version: unknown">
      <![CDATA[com.pulselms.app.framework.BrowserBotException: Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id='main']/div/div[2]/div[1]/h3
Build info: version: '2.47.1', revision: 'unknown', time: '2015-07-30 11:02:44'
System info: host: 'PC322939', ip: '10.25.154.48', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_71'
Driver info: driver.version: unknown
	at com.pulselms.app.framework.WebDriverBase.wait(WebDriverBase.java:89)
	at com.pulselms.webportal.page.DashboardPage.waitForDashboardPageToLoad(DashboardPage.java:29)
	at com.pulselms.webportal.test.LoginTest.validateLogin(LoginTest.java:68)
	at com.pulselms.webportal.test.LoginTest.pearsonPulseLoginTest(LoginTest.java:46)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- pearsonPulseLoginTest -->
  <testcase name="pearsonPulseLoginTest" time="29.252" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="62.056" classname="com.pulselms.webportal.test.LoginTest">
    <failure type="com.pulselms.app.framework.BrowserBotException" message="Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id=&#039;main&#039;]/div/div[2]/div[1]/h3
Build info: version: &#039;2.47.1&#039;, revision: &#039;unknown&#039;, time: &#039;2015-07-30 11:02:44&#039;
System info: host: &#039;PC322939&#039;, ip: &#039;10.25.154.48&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_71&#039;
Driver info: driver.version: unknown">
      <![CDATA[com.pulselms.app.framework.BrowserBotException: Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id='main']/div/div[2]/div[1]/h3
Build info: version: '2.47.1', revision: 'unknown', time: '2015-07-30 11:02:44'
System info: host: 'PC322939', ip: '10.25.154.48', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_71'
Driver info: driver.version: unknown
	at com.pulselms.app.framework.WebDriverBase.wait(WebDriverBase.java:89)
	at com.pulselms.webportal.page.DashboardPage.waitForDashboardPageToLoad(DashboardPage.java:29)
	at com.pulselms.webportal.test.LoginTest.validateLogin(LoginTest.java:68)
	at com.pulselms.webportal.test.LoginTest.pearsonPulseLoginTest(LoginTest.java:46)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- pearsonPulseLoginTest -->
  <testcase name="pearsonPulseLoginTest" time="27.747" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.366" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.599" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.320" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="26.064" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.503" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.678" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.477" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="24.754" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.801" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="23.284" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="26.836" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.913" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="31.050" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="59.515" classname="com.pulselms.webportal.test.LoginTest">
    <failure type="com.pulselms.app.framework.BrowserBotException" message="Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id=&#039;main&#039;]/div/div[2]/div[1]/h3
Build info: version: &#039;2.47.1&#039;, revision: &#039;unknown&#039;, time: &#039;2015-07-30 11:02:44&#039;
System info: host: &#039;PC322939&#039;, ip: &#039;10.25.154.48&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_71&#039;
Driver info: driver.version: unknown">
      <![CDATA[com.pulselms.app.framework.BrowserBotException: Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id='main']/div/div[2]/div[1]/h3
Build info: version: '2.47.1', revision: 'unknown', time: '2015-07-30 11:02:44'
System info: host: 'PC322939', ip: '10.25.154.48', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_71'
Driver info: driver.version: unknown
	at com.pulselms.app.framework.WebDriverBase.wait(WebDriverBase.java:89)
	at com.pulselms.webportal.page.DashboardPage.waitForDashboardPageToLoad(DashboardPage.java:29)
	at com.pulselms.webportal.test.LoginTest.validateLogin(LoginTest.java:68)
	at com.pulselms.webportal.test.LoginTest.pearsonPulseLoginTest(LoginTest.java:46)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- pearsonPulseLoginTest -->
  <testcase name="pearsonPulseLoginTest" time="30.121" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.110" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.477" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.479" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.959" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="21.158" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.090" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="31.504" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.468" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.051" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.905" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.882" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.724" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="31.131" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.055" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="23.536" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="27.524" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="31.485" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="22.951" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.179" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.027" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.261" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.895" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="31.296" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="26.667" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="30.528" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.826" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.878" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="49.040" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="31.190" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.384" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="33.976" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.586" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="29.661" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="28.383" classname="com.pulselms.webportal.test.LoginTest"/>
  <testcase name="pearsonPulseLoginTest" time="60.898" classname="com.pulselms.webportal.test.LoginTest">
    <failure type="com.pulselms.app.framework.BrowserBotException" message="Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id=&#039;main&#039;]/div/div[2]/div[1]/h3
Build info: version: &#039;2.47.1&#039;, revision: &#039;unknown&#039;, time: &#039;2015-07-30 11:02:44&#039;
System info: host: &#039;PC322939&#039;, ip: &#039;10.25.154.48&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_71&#039;
Driver info: driver.version: unknown">
      <![CDATA[com.pulselms.app.framework.BrowserBotException: Bot encountered a timeout while waiting for a condition,  Timed out after 30 seconds waiting for presence of element located by: By.xpath: //*[@id='main']/div/div[2]/div[1]/h3
Build info: version: '2.47.1', revision: 'unknown', time: '2015-07-30 11:02:44'
System info: host: 'PC322939', ip: '10.25.154.48', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_71'
Driver info: driver.version: unknown
	at com.pulselms.app.framework.WebDriverBase.wait(WebDriverBase.java:89)
	at com.pulselms.webportal.page.DashboardPage.waitForDashboardPageToLoad(DashboardPage.java:29)
	at com.pulselms.webportal.test.LoginTest.validateLogin(LoginTest.java:68)
	at com.pulselms.webportal.test.LoginTest.pearsonPulseLoginTest(LoginTest.java:46)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- pearsonPulseLoginTest -->
  <testcase name="pearsonPulseLoginTest" time="30.747" classname="com.pulselms.webportal.test.LoginTest"/>
</testsuite> <!-- com.pulselms.webportal.test.LoginTest -->
